### ISSUE TITLE
Windows on ARM build?

### ISSUE THREAD

**Issue by TreeBranches** [transmission-remote-gui/transgui#1475]
With these new Snapdragon X Elite PC's out there and reviewing well, is there an easy way to build a WinARM native app?

Windows 11.

I can provide assistance with testing if it helps.

**Comment by k0ste** [comment]
> I can provide assistance with testing if it helps.

I think you can try better luck here https://github.com/lighterowl/transgui/blob/develop/.github/build_windows.ps1

**Comment by lighterowl** [comment]
Not realistic as "aarch64-win64" isn't an official target for the Free Pascal Compiler. The only information I was able to find about this is [this mailing list announcement from 2020](https://lists.freepascal.org/pipermail/fpc-announce/2020-April/000614.html) and [this forum thread from 2023](https://forum.lazarus.freepascal.org/index.php?topic=61782.0). The announcement says :

> Simple programs work already and the compiler is at least able to display its help message (when trying to compiler something it silently  crashes, I've not yet found out why that is the case). Optimizations 
don't work yet, so please use -O- when building (especially when building the compiler/RTL).

The forum thread :

> FPC 3.3.1 supports aarch64-win64, but that target is not yet ready for prime time due to a bug, so you can't setup Lazarus for it as it wouldn't work.

To sum it up, it looks like support for this target never really went anywhere. I'll be glad to be proven wrong, though.
